docs/api/options.md

```diff
@@ -208,10 +208,13 @@ const wrapper = mount(WrapperComp).find(ComponentUnderTest)
 
 ## stubs
 
-- type: `{ [name: string]: Component | boolean } | Array<string>`
+- type: `{ [name: string]: Component | string | boolean } | Array<string>`
 
 Stubs child components. Can be an Array of component names to stub, or an object. If `stubs` is an Array, every stub is `<${component name}-stub>`.
 
+Depercation note:
+If declaring an object, string stubs are now deprecated and will be removed in a future version.
+
```

Comment:
I think we can just drop it entirely

Reply:
Because we dont have versioned docs, I think it makes sense to leave it. We are deprecating, but have not yet removed it. (We saw what removing `sync` did without mentioning it in the docs any more).

Maybe we can add an example what is now deprecated?